api-docs/cloud-load-balancers-v1/api-reference/methods/get-list-ciphers-v1.0-account-loadbalancers.rst

```diff
@@ -0,0 +1,111 @@
+.. _get-list-configured-ciphers:
+
+List ciphers
+~~~~~~~~~~~~~~~~~~~
+
+.. code::
+
+    GET /v1.0/{account}/loadbalancers/{loadbalancerid}/ssltermination/ciphers
+
+Lists load balancer ciphers that are configured for the load balancer.
+
+The response object shows a list of ciphers defined for the load balancer.
+
```

Comment:
Insert an intro sentence for the table, similar to "The following table shows the possible response codes for this operation:"